src/components/Dropdown.jsx

```diff
@@ -30,5 +31,15 @@ class Dropdown extends React.Component {
 		);
 	}
 }
+Dropdown.propTypes = {
+	name: PropTypes.string.isRequired,
+	label: PropTypes.string,
+	value: PropTypes.string,
+	options: PropTypes.shape({
+		name: PropTypes.string.isRequired,
+		value: PropTypes.string.isRequired,
+	}),
```

Comment:
Two problems here:

1) Options is an array, not an object. However, it's an array _of_ objects. This shape is good, but it should be wrapped in a different propType.
2) We iterate over the options array without checking for it first, so failing to pass one would cause an error. This should be a required propType.